client/app/helper/order-helper.service.spec.js

```diff
@@ -0,0 +1,90 @@
+describe("Service: OrderHelperService", () => {
+    let $q;
+    let $rootScope;
+    let $uibModal;
+
+    let OrderHelperService;
+
+    let deferred;
+
+    beforeEach(module("managerAppMock"));
+
+    beforeEach(inject((_$q_, _$rootScope_, _$uibModal_, _OrderHelperService_) => {
+        $q = _$q_;
+        $rootScope = _$rootScope_;
+        OrderHelperService = _OrderHelperService_;
+    }));
+
+    describe("transformToOrderValues", () => {
+        fit("should accept simple values", () => {
```

Comment:
If you merge you have to remove all `fit` in this file!
And than check if all tests pass.